DuckDuckGo/en.lproj/Localizable.strings

```diff
@@ -1405,8 +1408,9 @@ https://duckduckgo.com/mac";
 /* Title for the Windows waitlist button redirecting to Mac waitlist */
 "windows-waitlist.join-waitlist-screen.mac-waitlist" = "Looking for the Mac Version?";
 
-/* Title for the Join Windows Waitlist screen */
-"windows-waitlist.join-waitlist-screen.try-duckduckgo-for-windows" = "Get early access to try DuckDuckGo for Windows!";
+/* Title for the Join Windows Waitlist screen
+   Title for the Windows Waitlist feature */
+"windows-waitlist.join-waitlist-screen.try-duckduckgo-for-windows" = "Get DuckDuckGo for Windows!";
```

Comment:
Is this (and the change below) intentional? Perhaps we should keep the comment in one line.